### PR TITLE
Add sqrt function to myMath

### PR DESCRIPTION
**Pull request by fstehle**
PTAL: @timse

### CODE REVIEW

File: test/myMath.spec.js [hunk 1]
```diff
@@ -15,4 +15,11 @@ describe('MyMath', () => {
     });
 
   });
+
+  describe('sqrt', () => {
+    it('returns the correct square root', () => {
+      MyMath.sqrt(9).should.equal(3);
```
Comment (timse) [inline test/myMath.spec.js:21]: why do you test an external library?
but i guess same is to be said for the parse thingy :P

File: myMath.js [hunk 2]
```diff
@@ -1,13 +1,25 @@
...
+  static toInt(stringValue) {
+    return parseInt(stringValue, 10);
+  }
+
+  /**
+   * @param {number} number
+   * @return {number}
+   */
+  static sqrt(number) {
+    return mathjs.sqrt(number);
```
Comment (timse) [inline myMath.js:19]: `Math.sqrt` is a  native javascript function https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Math/sqrt

Comment (fstehle) [inline myMath.js:19]: I wanted to test inclusion of external libraries.

File: test/myMath.spec.js (hunk 1 above)
Comment (fstehle) [inline test/myMath.spec.js:21]: Because I can

**Pull request merged**